docs/extensibility/extending-blocks.md

```diff
@@ -20,6 +20,8 @@ function addListBlockClassName( settings, name ) {
 		return settings;
 	}
 
+	// Note: Object.assign() isn't supported in Edge, Andriod and mobile Opera
```

Comment:
Nitpick but we should have an Oxford comma here. 😄

Reply:
Edge supports `Object.assign`, but IE11 doesn't. The Edge/IE11 E-based icons appear similar in the table, but it's IE11 that lacks support.

Nitpick + 1: "Andriod" is a typo.

Thanks for adding this comment.

Reply:
Aside, I’m sure we use it in many other examples, too 

Should we promote extend from lodash to avoid those comments in the first okace?

Reply:
Is `_.extend()` functionally equivalent? https://underscorejs.org/#extend

Underscore is already bundled in WordPress and can be enqueued with `wp_enqueue_script( 'wp-utils' );`